semgrep-core/src/spacegrep/src/bin/Spacegrep_main.ml

```diff
@@ -24,6 +24,7 @@ type config = {
   time : bool;
   timeout : int option;
   warn : bool;
+  no_skip_search : bool;
```

Comment:
Although the CLI may expose `--no-skip-search` because you want this enabled by default, in the code I would have gone for `skip_search` since it's simpler to read. Minor comment.

Reply:
I'd rather not because "skip search" sounds like a request to skip the search, but it only means "skip search if possible".